aggregator/src/aggregator.rs

```diff
@@ -204,6 +199,10 @@ pub struct Config {
     /// becomes aware of key state changes.
     pub global_hpke_configs_refresh_interval: StdDuration,
 
+    /// Defines how long tasks should be cached for. This affects how often an aggregator becomes aware
+    /// of task parameter changes.
+    pub task_cache_ttl: Duration,
```

Comment:
It feels kind of bad to have a mix of messages::Duration and std::time::Duration here, but I really do need messages::Duration here so I can manipulate time with MockClock in tests. I kind of wish there was an std::time with a substitutable time source...